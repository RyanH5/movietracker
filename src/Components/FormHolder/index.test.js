import {FormHolder} from './index';
import { shallow } from 'enzyme';
import React from 'react';

describe('Login', () => {
  it('should match snapshot', () => {
    let wrapper = shallow(<FormHolder />);

    expect(wrapper).toMatchSnapshot();
  });
});