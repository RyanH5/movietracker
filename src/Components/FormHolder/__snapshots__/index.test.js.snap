// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login should match snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Unknown />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <div
          className="close-form-holder"
          onClick={[Function]}
>
          Ⓧ
</div>,
        <Connect(Login) />,
        <Connect(Signup) />,
      ],
      "className": "form-holder",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Ⓧ",
          "className": "close-form-holder",
          "onClick": [Function],
        },
        "ref": null,
        "rendered": "Ⓧ",
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {},
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {},
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <div
            className="close-form-holder"
            onClick={[Function]}
>
            Ⓧ
</div>,
          <Connect(Login) />,
          <Connect(Signup) />,
        ],
        "className": "form-holder",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Ⓧ",
            "className": "close-form-holder",
            "onClick": [Function],
          },
          "ref": null,
          "rendered": "Ⓧ",
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {},
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {},
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
